scripts/external_ips_scripts/create-cidr-block.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+ROX_ENDPOINT=${1:-localhost:8000}
+cidr_block=${2:-8.8.8.0/24}
+cidr_name=${3:-"testCIDR"}
+
+clusters_json="$(curl --location --silent --request GET "https://${ROX_ENDPOINT}/v1/clusters" -k --header "Authorization: Bearer $ROX_API_TOKEN")"
+
+cluster_id="$(echo "$clusters_json" | jq -r '.clusters[0].id')"
+
+cidr_json='{"entity": {"cidr": "'"$cidr_block"'", "name": "'"$cidr_name"'", "id": ""}}'
```

Comment:
[nit] to avoid the weird bash strings, and for some robustness we should probably use jq:

```suggestion
cidr_json="(jq -nc --arg cidr "$cidr_block" --arg name "$cidr_name" '{"entity": {"cidr": "$cidr", "name": "$name", "id": ""}}')"
```